### ISSUE TITLE
Missing Cart Types

### ISSUE THREAD

**Issue by zeekrey** [bigcommerce/storefront-data-hooks#47]
The Cart Type is missing the following type definition:

```
consignments: {
    available_shipping_options: {
      additional_description: string;
      cost: number;
      description: string;
      id: string;
      image_url: string;
      transit_time: string;
      type: string;
    }[];
  }[];
```

**Comment by jorgemasta** [comment]
Yes, we have to review all the types.

We could use a tool such as https://transform.tools/json-schema-to-typescript to transform the JSON Schema to TypeScript.

E.g. https://developer.bigcommerce.com/api-reference/store-management/carts/cart/getacart the schema can be taken from the **Responses** section

**Comment by zeekrey** [comment]
Do you accept PRs? Could find anything about contribution.

**Comment by jorgemasta** [comment]
Yes, we're open to PRs! 🙌 

Here you can a find a light contribution notes: https://github.com/bigcommerce/storefront-data-hooks/blob/master/CONTRIBUTING.md

I' m thinking of adding some notes on how to set up a local dev env, anything else you consider important?

**Comment by jorgemasta** [comment]
Solved in #29
